Revit_Core_Engine/Query/Bounds.cs

```diff
@@ -150,6 +150,34 @@ public static BoundingBoxXYZ Bounds(this IEnumerable<XYZ> points)
         }
 
         /***************************************************/
+
+        [Description("Returns combined bounding box of the given collection of bounding boxes.")]
+        [Input("bboxes", "A collection of the bounding boxes to get the bounds for.")]
+        [Output("bounds", "Combined bounding box of the bounding boxes.")]
+        public static BoundingBoxXYZ Bounds(this IEnumerable<BoundingBoxXYZ> bboxes)
+        {
+            List<XYZ> points = new List<XYZ>();
+
+            foreach (BoundingBoxXYZ bbox in bboxes)
+            {
+                Transform transform = bbox.Transform;
+
+                if (!transform.IsIdentity)
+                {
+                    points.Add(transform.OfPoint(bbox.Min));
+                    points.Add(transform.OfPoint(bbox.Max));
+                }
```

Comment:
Are you sure this is true? Shouldn't we use `BoundsOfTransformed` here?

Reply:
Yes, I will change it 👍